apps/website/docs/pipelines/azure-login.md

```diff
@@ -5,35 +5,72 @@ sidebar_label: Configuring Azure Login for GitHub Actions
 
 # Configuring Azure Login for GitHub Actions
 
-There are several ways to log into Azure by a GitHub Action. The DX chosen
-method doesn't require maintenance or secret management as it is a passwordless
-approach. Once set up, it just works.
+There are several ways to log into Azure using a GitHub Action. The DX-preferred
+method is passwordless, requiring no maintenance or secret management. Once set
+up, it works seamlessly.
 
-Only two components are needed to let GitHub Actions log into Azure:
+:::tip
+
+The modules `azure_federated_identity_with_github` and
+`azure_github_environment_bootstrap` already implement the steps described in
+this article. While the latter also creates the required GitHub Environments and
+secrets, both modules provide a default set of roles in the current Azure
+subscription, which is likely sufficient for new repositories (more on this
+later).
+
+:::
 
-1. An Azure Managed Identity resource
-1. A federation between the Azure Managed Identity and the GitHub repository
+To enable GitHub Actions to log into Azure, you need only two components:
 
-The folder `infra/identity` contains the Terraform definition to create both the
-Managed Identity and the federation with GitHub using a
-[Terraform custom module](https://github.com/pagopa/dx/tree/main/infra/modules/azure_federated_identity_with_github)
-. The module also provides a set of default roles in the current Azure
-subscription, which are likely to be enough for new repositories (more on this
-later...).
+1. A User-Assigned Azure Managed Identity resource
+2. A federation between the Azure Managed Identity and the GitHub repository
+
+Create a User-Assigned Managed Identity using the following command:
```

Comment:
suggestion: can we list the needed access rights in order to perform this operation?

Reply:
No specific roles are needed, you only need to have a write access

Reply:
to what? (I'd add this to the docs)